extensions/positron-python/python_files/positron/positron_ipykernel/ui.py

```diff
@@ -102,6 +102,9 @@ def on_comm_open(self, comm: BaseComm, msg: JsonRecord) -> None:
         self.browser = PositronViewerBrowser(comm=self._comm)
         webbrowser.register(self.browser.name, PositronViewerBrowser, self.browser, preferred=True)
 
+        # add patch to bokeh, if it is not imported
```

Comment:
Nit: What about placing this in `PositronIPyKernel` where we have some other patches to `warnings` and `holoviews`?